### DISCUSSION TITLE
Thingiverse page

### DISCUSSION THREAD

**Discussion by GalvanicGlaze** [Sakati84/3DPrintColorizer#13] [Ideas]
Hey , do you planning on adding this to thingiverse? 

It would be good for people to add remixes of mounts for different configurations.

Ill make my own so I can have the pen holder mount to a 4010 fan and i saw someone else mention an ender 3 v2 setup

**Comment by Sakati84** [comment]
Excellent idea! I will do this. Don‘t know why I didn‘t think of this myself though.

**Reply by Sakati84** [reply]
[https://www.thingiverse.com/thing:4918256](https://www.thingiverse.com/thing:4918256)

**Reply by GalvanicGlaze** [reply]
Nice, thanks 💖
